lib/sentry.dart

```diff
@@ -161,21 +159,23 @@ class SentryClient {
 
   /// Reports an [event] to Sentry.io.
   Future<SentryResponse> capture({@required Event event}) async {
-    final DateTime now = _clock.now();
+    final DateTime now = _clock();
+    String authHeader = 'Sentry sentry_version=6, sentry_client=$sentryClient, '
+        'sentry_timestamp=${now.millisecondsSinceEpoch}, sentry_key=$publicKey';
+    if (secretKey != null) {
+      authHeader += ', sentry_secret=$secretKey';
```

Comment:
Let's add a unit-test for this case.

Reply:
Added unit test for parsing and http header.